src/fsharp/FSharp.Compiler.Service.ProjectCracker.netcore/ProjectCrackerTool.fs

```diff
@@ -0,0 +1,361 @@
+﻿namespace Microsoft.FSharp.Compiler.SourceCodeServices.ProjectCrackerTool
```

Comment:
This looks more or less duplicated from https://github.com/fsharp/FSharp.Compiler.Service/blob/340b9ab6ca1dd90ab7fcd89d503acc4553a3bca9/src/fsharp/FSharp.Compiler.Service.ProjectCrackerTool/Program.fs.  Do you think we can avoid that duplication?